mainWidget/main_widget_FloorsAndRooms.yaml

```diff
@@ -51,8 +51,9 @@ slots:
                           default:
                             - component: widget:main_widget_Light_Card
                               config:
-                                Title: "=loop.lightItem.metadata.uiSemantics ? loop.lightItem.metadata.uiSemantics.config.equipment + loop.lightItem.metadata.uiSemantics.config.preposition + loop.lightItem.metadata.uiSemantics.config.location : loop.lightItem.label"
+                                Title: "=loop.lightPowerItem.metadata.uiSemantics ? loop.lightPowerItem.metadata.uiSemantics.config.equipment + loop.lightPowerItem.metadata.uiSemantics.config.preposition + loop.lightPowerItem.metadata.uiSemantics.config.location : loop.lightItem.label"
                                 groupItem: =loop.lightItem.name
+                                groupCategory: =loop.lightItem.category
                                 powerItem: = loop.lightPowerItem.name
```

Comment:
Why introducing an new param to pass to the LightCard ?

Reply:
Os sorry that was in intermediate solution only.

At the end it turned out that the title just needed to be changed to make it work.